Pipelines/build-deploy-prod-SampleSolution.yml

```diff
@@ -3,13 +3,12 @@ resources:
   repositories:
   - repository: PipelineRepo  # repository name (DO NOT CHANGE THIS VALUE) 
     type: git
-    ref: BranchContainingTheBuildTemplates # If your pipeline templates are in a branch other than the default branch specify the branch here. Otherwise the default branch will be used by default.
+    ref: BranchContainingTheBuildTemplates  # If your pipeline templates are in a branch other than the default branch specify the branch here. Otherwise the default branch will be used by default.
     name: RepositoryContainingTheBuildTemplates  #This is the name of the repo in the current project in Azure Devops that has the pipeline templates. If the repo is in a different project you can specify the project and repo using the format ProjectContainingTheBuildTemplates/RepositoryContainingTheBuildTemplates (https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#type)
 trigger:
   branches:
     include:
     # Replace the following with actual branch name(s) in your repo for which you want to trigger a build. The assumption here is that you only want to only trigger on a change to a specific branch rather than a change on any branch which would include your working branch for development.'
-    - SampleSolutionName-main
```

Comment:
Should be
- SampleSolutionName

For consistency between templates

Reply:
@jenschristianschroder that line was removed from the updated file. So, it should be good.

Reply:
Sorry, i wasn't clear with my comment :) 
I mean the removed line should be reinserted and modified to SampleSolutionName.

I think it must also trigger on SampleSolutionName branch. The other pipelines do.

Reply:
That pipeline was updated to be the production release pipeline so it only triggers on main now. Test triggers on the Solution branch and Validation no longer has any triggers since it's triggered by a branch policy.

Reply:
Oops no I lied. Validation triggers on refs/pull/* but it should be correct as is. Thanks @jenschristianschroder!